event-schema/event_schema.go

```diff
@@ -839,9 +863,12 @@ func assertError(err error) {
 
 func assertTxnError(err error, txn *sql.Tx) {
 	if err != nil {
-		txn.Rollback()
-		pkgLogger.Info(fmt.Sprintf("%#v\n", err))
-		pkgLogger.Info(fmt.Sprintf("%#v\n", txn))
+		pkgLogger.Error(err.Error())
+		pkgLogger.Debugf("%+v", txn)
+		if rollbackErr := txn.Rollback(); rollbackErr != nil {
+			pkgLogger.Error(err.Error())
+			panic(fmt.Errorf("%v: %v", err, rollbackErr))
```

Comment:
IMO failing to rollback a transaction, most probably, wouldn't have any detrimental effect to the system, since the most common scenario would be that the transaction is already rolled back. Based on the above we could ignore this error (just log it).